tests/test_pixel.py

```diff
@@ -771,15 +780,16 @@ def test_single_image():
             },
         },
     }
-    if __name__ == '__main__':
+    if __name__ != '__main__':
+        config['verbose'] = 0
+    if True:
```

Comment:
I was flipping which sets of tests to include always and which to include only from the command line (not from nosetests) to try to get full test coverage while still not taking forever.  But I'm happy to get rid of the `if True` and slide things left now.